testing/kuttl/e2e/pgbouncer/01-psql-connect.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-connect
+spec:
+  template:
+    spec:
+      backoffLimit: 3
+      restartPolicy: "OnFailure"
+      containers:
+        - name: psql
+          securityContext:
+            allowPrivilegeEscalation: false
+            privileged: false
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+          command:
+            - "bash"
+            - "-c"
+            # Using `install` to move the files/change the ownership for psql
+            - "install -D /tmp/certs/ca.crt /tmp/certs/tls.crt /tmp/certs/tls.key -t /pgdata/certs -m 600 && psql -c 'select version();'"
+          env:
+          - name: PGSSLMODE
+            value: verify-full
+          - name: PGSSLCERT 
+            value: "/pgdata/certs/tls.crt"
+          - name: PGSSLKEY
+            value: "/pgdata/certs/tls.key"
+          - name: PGSSLROOTCERT
+            value: "/pgdata/certs/ca.crt"
```

🔧 PgBouncer is not configured to verify client certificates, so there's no need to present one here. To successfully `verify-full` the client needs only the CA, and that file doesn't need any special permissions.

I believe we can run `psql` without copying the certificate files.

```suggestion
          - name: PGSSLROOTCERT
            value: "/tmp/certs/ca.crt"
```

---

Ah, ok, tested and verified, so I could make this test simpler